app/src/main/java/com/android/partagix/ui/App.kt

```diff
@@ -223,12 +223,15 @@ class App(
       composable(
           Route.ACCOUNT,
       ) {
-        ViewAccount(navigationActions = navigationActions, userViewModel = UserViewModel())
+        userViewModel.setUserToCurrent()
```

Comment:
I think that it is not necessary (or a duplication) because in the init of UserViewModel, setUserToCurrent is already called.

Reply:
This part is important, when we modify the user it needs to reupdate before going back, also it's not very expensive as it's called only once